signalfx/client.py

```diff
@@ -61,23 +62,48 @@ def send_event(self, event_type, category=None, dimensions=None,
         logging.debug('Sending event to SignalFx: %s', data)
         return data
 
+    @staticmethod
+    def update_metric(metric_type, description=None,
+                      custom_properties=None, tags=None):
+        data = {'type': metric_type.upper(),
+                'description': description or '',
+                'customProperties': custom_properties or {},
+                'tags': tags or []}
+        logging.debug('Sending metric metadata to SignalFx: %s', data)
+        return data
+
+    @staticmethod
+    def update_tag(description=None, custom_properties=None):
+        data = {'description': description or '',
+                'customProperties': custom_properties or {}}
+        logging.debug('Sending tag data to SignalFx: %s', data)
+        return data
+
 
 class SignalFxClient(BaseSignalFx):
     """SignalFx API client.
 
     This class presents a programmatic interface to SignalFx's metadata and
-    ingest APIs. At the time being, only ingest is supported; more will come
-    later.
+    ingest APIs. At the time being, ingest and retrieval of data
```

Comment:
NIT: Weird sentence structure "At the time being,..." At the moment?  For the time being?